srcpkgs/paraview/template

```diff
@@ -0,0 +1,64 @@
+# Template file for 'paraview'
+pkgname=paraview
+version=5.6.1
+revision=1
+archs="x86_64"
+wrksrc=ParaView-v${version}
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release
```

Comment:
`-DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr` are already set by the build style (see `common/build-style/cmake.sh`). No need to add them here.

Reply:
Thanks Johnny, changed in 3c065ef682